torch/csrc/jit/serialization/pickler.cpp

```diff
@@ -480,6 +480,20 @@ void Pickler::pushLiteralTensor(const IValue& ivalue) {
   // Construct the collections.OrderedDict for the backward_hooks
   push<PickleOpCode>(PickleOpCode::REDUCE);
 
+  if (!quantized) {
```

Comment:
also not sparse right?

Reply:
Sparse is handled by different function. 

https://github.com/pytorch/pytorch/blob/0ae4ac2b6cfd7c35637c3dcd49e26536d0d9b08a/torch/csrc/jit/serialization/pickler.cpp#L357

(Only Quantized and regular Tensor share this function)